### PR TITLE
Template Codeblock with example for index.vue

### PR DESCRIPTION
**Pull request by diabolical17**
Adds code blocks to replace GIFs of code and a template for documentation code.

### CODE REVIEW

**Comment by diabolical17** [comment]
Honestly, I don't really know why I created this PR. Its the tiny little things 🤣

**Comment by falseresync** [comment]
May you look into animating that? Also, send a picture please

**Comment by diabolical17** [comment]
I consider my changes complete. Final Result:

**Comment by diabolical17** [comment]
Ok now its done 😅

**Comment by diabolical17** [comment]
Sorry, going to finish this PR as i've been side-tracked.

**Comment by diabolical17** [comment]
Final (With syntax highlighting edits from discord)

**Comment by diabolical17** [comment]
Template:

```vue
<pre v-highlightjs>
<code class="javascript">Javascript code goes here.</code>
        </pre>
```

File: pages/index.vue [hunk 1]
```diff
@@ -193,6 +202,8 @@ export default {
 </script>
 
 <style lang="scss" scoped>
+@import url('https://fonts.googleapis.com/css2?family=Fira+Code&display=swap');
```
Comment (falseresync) [inline pages/index.vue:205]: nope, no fonts. bad idea.

**Review by falseresync — CHANGES_REQUESTED** [review]

File: pages/index.vue (hunk 1 above)
Comment (diabolical17) [reply pages/index.vue:205]: What font would you like me to set the inner code block? monospace?

Comment (falseresync) [reply pages/index.vue:205]: should be good enough, or you can use `Consolas, "Andale Mono WT", "Andale Mono", "Lucida Console", "Lucida Sans Typewriter", "DejaVu Sans Mono", "Bitstream Vera Sans Mono", "Liberation Mono", "Nimbus Mono L", Monaco, "Courier New", Courier, monospace;` stack

Comment (diabolical17) [reply pages/index.vue]: Fixed.

**Pull request merged**